src/endpoint/s3/s3_utils.js

```diff
@@ -322,6 +322,21 @@ function set_response_object_md(res, object_md) {
     }
 }
 
+/**
+ * @param {nb.S3Response} res 
+ */
+function set_response_supported_storage_classes(res) {
+    const storage_classes = [];
+    if (!config.DENY_UPLOAD_TO_STORAGE_CLASS_STANDARD) {
+        storage_classes.push('STANDARD');
+    }
+    if (config.NSFS_GLACIER_ENABLED && config.NSFS_GLACIER_LOGS_ENABLED) {
+        storage_classes.push('GLACIER');
+    }
+
+    res.setHeader('x-noobaa-available-storage-classes', storage_classes);
```

Comment:
I know it's a one appearance, but for headers we return this is like an API so I would prefer to set them as const
```js
const X_NOOBAA_AVAILABLE_STORAGE_CLASSES = 'x-noobaa-available-storage-classes';
...
    res.setHeader(X_NOOBAA_AVAILABLE_STORAGE_CLASSES, storage_classes);
```